functionnal_tests/cypress/integration/publication/publication_vue_recent_activities_spec.js

```diff
@@ -1,38 +1,40 @@
-import { PAGES } from '../../support/urls_commands'
+// 2021-09-06 - MB - Unstable test. See : https://github.com/tracim/tracim/issues/4905
```

Comment:
Prefer using .skip instead of commenting tests.

Reply:
Thanks, I will fix this this